test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -268,7 +269,7 @@ public void MobileTables()
                 log => log.FormattedMessage.Contains("Exports: IsObject=true, Count=1"));
         }
 
-        [Fact(Skip = "Needs investigation")]
```

Comment:
While I was in here, I reenabled all the tests I could. There are a small set remaining now, which require fixes in the framework to enable